views/proposals.tmpl

```diff
@@ -118,12 +118,38 @@
                     <tr>
                         <td class="text-left"><a href="/proposal/{{.RefID}}">{{.Name}}</a></td>
                         <td class="text-left">{{.Username}}</td>
-                        {{with .Status}}
-                            <td class="text-left">{{toTitleCase .String}}</td>
-                        {{end}}
                         <td class="text-left">
-                            {{if .VoteStatus}}{{with .VoteStatus}}{{.ShortDesc}}{{end}}{{else}}N/A{{end}}
+                            {{if .VoteStatus}}
+                                {{range $i, $vr := .VoteResults}}
+                                    {{if eq $vr.Option.OptionID "yes"}}
+                                        {{$votesPercent :=  percentage $vr.VotesReceived $v.TotalVotes}}
+                                        {{if lt $.Tip.Height (toInt $v.Endheight)}}
+                                            <span style="color:#2970ff;">
```

Comment:
As a general rule, no inline styles in the templates. Some of these text colors may already exists as a class defintion, otherwise they should be created, New text coloring classes can be defined in `public/scss/typograghy.scss`. You can also add a `body.darkBG` descendant selector in `themes.scss` if you need. See the class `text-green` as an example. 

I would recommend having a dark mode color for the `Abandoned` proposals, as the look isn't great with `#091440` in both modes.

Reply:
I'm working on the styling now